app/helpers/coronavirus_timeline_nations_helper.rb

```diff
@@ -0,0 +1,50 @@
+module CoronavirusTimelineNationsHelper
+  UK_COUNTRY_LIST = %w[england northern_ireland scotland wales].freeze
+
+  def show_timeline_nations?(timeline_list)
+    timeline_list.any? { |item| item["national_applicability"] }
+  end
+
+  def timeline_for_nation(timeline_list, nation)
+    timeline_list.select { |item| item["national_applicability"].include?(nation) }
+  end
+
+  def display_country(selected_country = "england")
+    return "england" unless UK_COUNTRY_LIST.include?(selected_country)
+
+    selected_country
+  end
+
+  def timeline_nations_items(selected_country = nil)
+    selected_country = "england" unless UK_COUNTRY_LIST.include?(selected_country)
+
+    UK_COUNTRY_LIST.map do |value|
+      {
+        value: value,
+        text: value.titleize,
+        checked: selected_country == value,
+        data_attributes: {
+          track_category: "pageElementInteraction",
+          track_action: "TimelineNation",
+          track_label: value.titleize,
+        },
+      }
+    end
+  end
+
+  def timeline_nation_tags(national_applicability)
+    if uk_wide?(national_applicability)
+      "<strong class='govuk-tag govuk-tag--blue'>UK Wide</strong>"
+    else
+      nation_tags = national_applicability.map do |nation|
+        "<strong class='govuk-tag govuk-tag--blue'>#{nation.titleize}</strong>"
+      end
+
+      nation_tags.join(" ")
+    end
+  end
+
+  def uk_wide?(national_applicability)
+    UK_COUNTRY_LIST.sort == national_applicability.uniq.sort
+  end
```

Comment:
Nitpick: this would be better suited to being a variable than a helper method given this is a method is only used in one place, doesn't need public visibility and would be globally available despite only a niche context.

```
uk_wide = UK_COUNTRY_LIST.sort == national_applicability.uniq.sort
if uk_wide
```